README.md

```diff
@@ -14,12 +14,10 @@ To start setting up the environment with BDD (Behavior Driven Development).
 
 ## Configuration
 List of required libraries in Project interpreter
-* Install [behave](https://behave.readthedocs.io/en/stable/) interpreter in PyCharm.
+* Install [behave](https://www.jetbrains.com/help/pycharm/creating-feature-files.html) interpreter in PyCharm.
 * Install [requests](https://pypi.org/project/requests/)
 * Install [jsonschema](https://pypi.org/project/jsonschema/)
 * Install [Pytest](https://docs.pytest.org/en/latest/getting-started.html) to write unit tests
```

Comment:
all of this we install through pycharm

Reply:
yes because is the configuration of the IDE

Reply:
But it would show the procedure in the IDE, not like a link that needs to be downloaded